pkg/minishift/addon/command/oc_command.go

```diff
@@ -28,20 +28,24 @@ type OcCommand struct {
 	*defaultCommand
 }
 
-func NewOcCommand(command string) *OcCommand {
-	defaultCommand := &defaultCommand{rawCommand: command}
+func NewOcCommand(command string, ignoreError bool) *OcCommand {
+	defaultCommand := &defaultCommand{rawCommand: command, ignoreError: ignoreError}
 	ocCommand := &OcCommand{defaultCommand}
 	defaultCommand.fn = ocCommand.doExecute
 	return ocCommand
 }
 
-func (c *OcCommand) doExecute(ec *ExecutionContext) error {
+func (c *OcCommand) doExecute(ec *ExecutionContext, ignoreError bool) error {
 	// split off the actual 'oc' command. We are using our cached oc version to run oc commands
 	cmd := strings.Replace(c.rawCommand, "oc ", "", 1)
 	cmd = ec.Interpolate(cmd)
 	fmt.Print(".")
 
 	commander := ec.GetOcCommander()
+	if ignoreError {
+		commander.Run(ec.Interpolate(cmd), ioutil.Discard, ioutil.Discard)
```

Comment:
Do we want the error to be shown if run with logging enabled?

Reply:
@anjannath this complete command interface stuff doesn't have any logger, that's another reason we are sending stdin/stderr to `ioutil.Discard`.

Reply:
I see.. maybe we can do something about it after #169 is resolved

Reply:
@praveenkumar You could have ignored the exitStatus too instead of calling the function with ioutil.Discard for stderr. Why did you choose to call the `Run` separately?

Reply:
@LalatenduMohanty If you not call Run then command execution doesn't going to happen, also ignoring exitStatus mean we are just ignoring the command exit status but if you look how `Run` works then it put stderr to stdin so only way to make it work the way @jorgemoralespou wants to redirect stderr => dev/null which ioutil.Discard do.

Reply:
Right, I modified the code and saw this
```
$ minishift addon apply admin-user
-- Applying addon 'admin-user':.Error from server (AlreadyExists): users.user.openshift.io "admin" already exists
.
```